### DISCUSSION TITLE
Switching from Hardrock/Easy to difficulty adjust should automatically change difficulty adjust to the equivalent settings

### DISCUSSION THREAD

**Discussion by Natelytle** [ppy/osu#17175] [Ideas] [answered]
If I want to change a difficulty setting while playing with HR/EZ on Lazer, I am forced to dance around with HR/EZ and DA inputting each of the settings one by one. I feel like it would feel more natural if it just automatically changed the difficulty settings upon switching instead of resetting them all. Alternatively you could have difficulty adjust "presets" that you can select, but that seems unnecessary and Lazer has enough toggles as is.

**Comment by bdach** [comment, marked as answer]
mod presets are present on the new mod overlay design which is a work in progress (https://www.figma.com/file/NkdREXr6wFkA99oLw8PnBv/Client%2FMod-Selection?node-id=0%3A17)
